sky/skylet/events.py

```diff
@@ -129,15 +129,20 @@ def _stop_cluster(self, autostop_config):
             config = common_utils.read_yaml(self._ray_yaml_path)
 
             provider_module = config['provider']['module']
-            provider_search = re.search(r'(?:providers|provision)\.(.*)(\.)?',
+            # Examples:
+            #   'sky.skylet.providers.aws.AWSNodeProviderV2' -> 'aws'
+            #   'sky.provision.aws' -> 'aws'
+            provider_search = re.search(r'(?:providers|provision)\.(\w+)\.?',
                                         provider_module)
             assert provider_search is not None, config
             provider_name = provider_search.group(1).lower()
-
-            if provider_name in ['aws', 'gcp']:
+            if provider_name in ('aws', 'gcp'):
+                logger.debug('Using new provisioner to stop the cluster.')
```

Comment:
Let’s use logger.info instead to print it to skylet.log by default. This is not directly user facing, so info should be good.